src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1330,8 +1330,8 @@ else if (path.toFile().getName().contains("sun")) {
             styleChecks.remove("TreeWalker");
             styleChecks.remove("Checker");
 
-            Assert.assertTrue(fileName + " requires the following check(s) to appear: "
-                    + styleChecks, styleChecks.isEmpty());
+            //Assert.assertTrue(fileName + " requires the following check(s) to appear: "
```

Comment:
unless you are going to fix them in this issue, you should probably create a new issue and comment `until ...`

Reply:
comment was added